logstream.go

```diff
@@ -17,6 +17,15 @@ type LogStream struct {
 	service *cloudwatchlogs.CloudWatchLogs
 }
 
+func Filter(vs []*cloudwatchlogs.LogStream, f func(*cloudwatchlogs.LogStream) bool) []*cloudwatchlogs.LogStream {
```

Comment:
This should probably be lowercase as we don't want to export it from the package.